ethportal-api/src/types/execution/header.rs

```diff
@@ -277,7 +277,7 @@ impl From<Header> for RpcHeader {
             excess_blob_gas: excess_blob_gas.map(|v| v.to()),
             hash,
             parent_beacon_block_root: parent_beacon_block_root.map(|h264| h264.0.into()),
-            total_difficulty: Some(difficulty),
+            total_difficulty: None,
```

Comment:
Could you just add a comment here explaining why this is set to `None`

Reply:
I think they are still part of the accumulators (based on quick look [here](https://github.com/ethereum/trin/blob/4692e6bde873243884f3820a8b417616efca26c9/trin-validation/src/accumulator.rs)), but they are not part of the content value object.

So yes, you are right that we don't have access to them.
If block is post-merge, we could set it as value is now always the same (basically have hardcoded value), but since we don't have it pre-merge, I think it's better to just say it's unsupported.